loopchain/baseservice/peer_manager.py

```diff
@@ -116,46 +80,54 @@ def __init__(self, channel_name):
         """
         self.peer_list_data = PeerListData()
         self.__channel_name = channel_name
-        self.__peer_object_list = {}
+        self.peer_object_list = {}
 
-        # peer group 은 전체 peer 가 등록된 ALL GROUP 과 개별 group_id 로 구분된 리스트가 존재한다.
-        self.__init_peer_group(conf.ALL_GROUP_ID)
         # lock object for if add new peer don't have order that must locking
         self.__add_peer_lock: threading.Lock = threading.Lock()
 
         self.__leader_complain_count = 0
         self.__highest_block_height = -1    # for RS heartbeat
 
-    @property
-    def peer_object_list(self) -> dict:
+    def peer_ids_hash(self):
```

Comment:
Is it replaced with rep hash?

Reply:
Replacing repHash will be updated at the next pull request.